src/datepicker/types.js

```diff
@@ -102,19 +102,19 @@ export type CalendarInternalState = {
 
 export type CalendarPropsT = {
   /** Defines if the calendar is set to be focused on an initial render. */
-  calFocusedInitially: boolean,
+  autoFocuseCalendar: boolean,
```

Comment:
should it be `autoFocusCalendar`? or maybe simply `autoFocus`

Reply:
I'd keep the `Calendar` part since it makes a date in the calendar focused and not the input element of the datepicker. So the `autoFocus` may be confusing

Reply:
oh, okay, that makes perfect sense - thanks for the clarification!